source/includes/inventory/login.md

```diff
@@ -0,0 +1,20 @@
+# Authentication
+
+In order to authentify yourself on the application, you need to visit this [website](https://google.fr). It will generate a QRCode that contains a token set to expire after 24 hours.
+
+<aside class="notice">
+  You can only login with a QRCode.
+</aside>
+
+Once you have obtained a QRCode, scan it with the application.
+
+![login](images/login.png)
+
+If the token is valid you should be authentified and redirected to the application's home.
+
+If not, a red banner saying **USER NOT FOUND** will appear briefly.
+
+![login-error](images/login-error.png)
```

Comment:
the documentation is shared for all our projects/applications, please use a dedicated folder for this particular application otherwise it will be a nightmare to know which images goes to which project (moreover login-error is not self explanatory)